contracts/executable/AxelarGMPExecutable.sol

```diff
@@ -61,7 +61,8 @@ abstract contract AxelarGMPExecutable is IAxelarGMPExecutable {
         bytes32 commandId,
         string calldata sourceChain,
         string calldata sourceAddress,
-        bytes calldata payload
+        bytes memory payload,
+        bytes32 payloadHash
```

Comment:
other apps don't need the payload hash, we might have to take the gas penalty in ITS